guide/14-deep-learning/how_retinanet_works.ipynb

```diff
@@ -0,0 +1,159 @@
+{
```

Comment:
<p>There are four major components of a RetinaNet model architecture <strong>(Figure x)</strong>:</p><p>What is figure x. Since I can't see images, not sure if it is labelled somewhere.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Typo: It's supposed to be 'Figure 3'</p>

Reply:
<p>Typo: It's supposed to be 'Figure 3'</p>

Reply:
<p>It's the retinanet architecture image.</p>